docs/model_library/operational_water_management/index.rst

```diff
@@ -878,89 +878,89 @@ Assumptions:
 
 The water quality at nodes is dependent on the flow rates into the node and the water quality of the flows. Even mixing is assumed, so all outgoing flows have the same water quality.
 
 .. math::
 
-    \sum\nolimits_{(p,n)∈PNA}\textcolor{purple}{F_{l,l,t}^{Piped}}⋅\textcolor{red}{Q_{p,w,t}} +\sum\nolimits_{(p,n)∈CNA}\textcolor{purple}{F_{l,l,t}^{Piped}}⋅\textcolor{red}{Q_{p,w,t}}
+    \sum\nolimits_{(p,n)\in PNA}\textcolor{purple}{F_{l,l,t}^{Piped}} \cdot \textcolor{red}{Q_{p,w,t}} +\sum\nolimits_{(p,n)\in CNA}\textcolor{purple}{F_{l,l,t}^{Piped}} \cdot \textcolor{red}{Q_{p,w,t}}
 
-    +\sum\nolimits_{(n ̃,n)∈NNA}\textcolor{purple}{F_{l,l,t}^{Piped}}⋅\textcolor{red}{Q_{n,w,t}}+\sum\nolimits_{(s,n)∈SNA}\textcolor{purple}{F_{l,l,t}^{Piped}}⋅\textcolor{red}{Q_{s,w,t}}
+    +\sum\nolimits_{(\tilde{n},n)\in NNA}\textcolor{purple}{F_{l,l,t}^{Piped}} \cdot \textcolor{red}{Q_{n,w,t}}+\sum\nolimits_{(s,n)\in SNA}\textcolor{purple}{F_{l,l,t}^{Piped}} \cdot \textcolor{red}{Q_{s,w,t}}
 
-    = \textcolor{red}{Q_{n,w,t}}⋅(\sum\nolimits_{(n,n ̃)∈NNA}\textcolor{purple}{F_{l,l,t}^{Piped}} +\sum\nolimits_{(n,p)∈NCA}\textcolor{purple}{F_{l,l,t}^{Piped}}
+    = \textcolor{red}{Q_{n,w,t}} \cdot (\sum\nolimits_{(n,\tilde{n})\in NNA}\textcolor{purple}{F_{l,l,t}^{Piped}} +\sum\nolimits_{(n,p)\in NCA}\textcolor{purple}{F_{l,l,t}^{Piped}}
 
-    +\sum\nolimits_{(n,k)∈NKA}\textcolor{purple}{F_{l,l,t}^{Piped}} +\sum\nolimits_{(n,r)∈NRA}\textcolor{purple}{F_{l,l,t}^{Piped}}
+    +\sum\nolimits_{(n,k)\in NKA}\textcolor{purple}{F_{l,l,t}^{Piped}} +\sum\nolimits_{(n,r)\in NRA}\textcolor{purple}{F_{l,l,t}^{Piped}}
 
-    +\sum\nolimits_{(n,s)∈NSA}\textcolor{purple}{F_{l,l,t}^{Piped}} +\sum\nolimits_{(n,o)∈NOA}\textcolor{purple}{F_{l,l,t}^{Piped}})
+    +\sum\nolimits_{(n,s)\in NSA}\textcolor{purple}{F_{l,l,t}^{Piped}} +\sum\nolimits_{(n,o)\in NOA}\textcolor{purple}{F_{l,l,t}^{Piped}})
 
 
-**Beneficial Reuse Water Quality** ∀o ∈ O, w ∈ W, t ∈ T
+**Beneficial Reuse Water Quality** :math:`\forall o \in O, w \in W, t \in T`
 
 The water quality at beneficial reuse sites is dependent on the flow rates into the site and the water quality of the flows.
 
 .. math::
 
-    \sum\nolimits_{(n,o)∈NOA}\textcolor{purple}{F_{l,l,t}^{Piped}}⋅\textcolor{red}{Q_{n,w,t}} +\sum\nolimits_{(s,o)∈SOA}\textcolor{purple}{F_{l,l,t}^{Piped}}⋅\textcolor{red}{Q_{s,w,t}} +\sum\nolimits_{(p,o)∈POT}\textcolor{purple}{F_{l,l,t}^{Trucked}}⋅\textcolor{red}{Q_{p,w,t}}
+    \sum\nolimits_{(n,o)\in NOA}\textcolor{purple}{F_{l,l,t}^{Piped}} \cdot \textcolor{red}{Q_{n,w,t}} +\sum\nolimits_{(s,o)\in SOA}\textcolor{purple}{F_{l,l,t}^{Piped}} \cdot \textcolor{red}{Q_{s,w,t}} +\sum\nolimits_{(p,o)\in POT}\textcolor{purple}{F_{l,l,t}^{Trucked}} \cdot \textcolor{red}{Q_{p,w,t}}
 
-    = \textcolor{red}{Q_{o,w,t}}⋅\textcolor{purple}{F_{o,t}^{BeneficialReuseDestination}}
+    = \textcolor{red}{Q_{o,w,t}} \cdot \textcolor{purple}{F_{o,t}^{BeneficialReuseDestination}}
 
 
 .. _model_terminology:
```

Comment:
For more extra credit, change line 981 to

    :math:`[\textcolor{blue}{t}]` **or** :math:`[\textcolor{blue}{t \in T}]` **:** This notation indicates that timing of capacity expansion has not yet been implemented.